tests/baselines/reference/typeFromPropertyAssignment17.errors.txt

```diff
@@ -0,0 +1,32 @@
+tests/cases/conformance/salsa/minimatch.js(1,1): error TS2304: Cannot find name 'module'.
+tests/cases/conformance/salsa/use.js(1,12): error TS2304: Cannot find name 'require'.
+
+
+==== tests/cases/conformance/salsa/use.js (1 errors) ====
+    var mini = require('./minimatch')

```

Comment:
maybe add a `.d.ts` into the test that declares `require` and `module`, just so there's no unnecessary error baseline?

Reply:
I tried that already, and, because of the uses of require/exports, these files aren't in the global namespace any more. I forgot that I could use triple-slash references to work around that.

I still couldn't get rid of the error on assignment to exports.